Sources/Commands/SwiftPackageTool.swift

```diff
@@ -1967,3 +1983,18 @@ extension BuildOperation {
         }
     }
 }
+
+extension OutputByteStream {
```

Comment:
This is currently copied from the BuildOperation where it is private.  We should have only one implementation, so I'm looking for opinions on where this code should live.

Reply:
Move this down to Basics.  I think it's good actually that there is a public accessor for checking whether a stream is connected to a TTY.